media-gfx/curaengine/curaengine-0.15.04.6.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+MY_PV=${PV#0.}
+MY_PN=CuraEngine
+
+SRC_URI="https://github.com/Ultimaker/${MY_PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+DESCRIPTION="A 3D model slicing engine for 3D printing"
+HOMEPAGE="https://github.com/Ultimaker/CuraEngine"
+
+LICENSE="AGPL-3"
+SLOT="0"
+IUSE="test"
+
+RDEPEND=""
+DEPEND=""
+
+S="${WORKDIR}/${MY_PN}-${MY_PV}"
+PATCHES=( "${FILESDIR}"/${P}-cflags.patch)
```

Comment:
This is very nitpicky but maybe you should leave a space before ')' since you have one after '(' :)